modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java

```diff
@@ -553,6 +556,15 @@ private void _validateActive(
 		}
 	}
 
+	private void _validateDescription(String description)
+		throws ObjectActionDescriptionException {
```

Comment:
@Aquiiiles please throw `PortalException` like other validation methods in this class